### DISCUSSION TITLE
View the region/ country a report was submitted from?

### DISCUSSION THREAD

**Discussion by Elliot-12345** [globaleaks/globaleaks-whistleblowing-software#3709] [Q&A]
Is there a way to view the region/ country a report was submitted from? I realise the primary objective of the application is anonymity, but being able to see this information will allow our team to make informed decisions on how best to move forward with a report.

**Comment by evilaliv3** [comment]
Thank you @Elliot-12345; This kind of tracking is actually not considered in this application.

It is up to whistleblowers to decide if and what they want to tell you.

In which jurisdiction are you operating?

Any tracking of this kind in a whistleblowing platform is probably banned in EU.
